docker/docker-in-docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jenkins:latest
+MAINTAINER edxops
+
+USER root
+RUN apt-get update \
+      && apt-get install -y sudo python-dev python-pip libmysqlclient-dev apt-transport-https ca-certificates \
+      && rm -rf /var/lib/apt/lists/*
+RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+COPY docker.list /etc/apt/sources.list.d/docker.list
+RUN apt-cache policy docker-engine
```

Comment:
You don't need to run this as a part of the build.  It was just in the instructions for visual verification.